server/db.js

```diff
@@ -0,0 +1,20 @@
+  
+require('dotenv').config();
+const { Pool } = require('pg')
+
+const user = process.env.PG_USERNAME || "postgres";
+const password = process.env.PG_PASSWORD || "postgres";
+const host = process.env.PG_HOST || "localhost";
+const dbName = process.env.PG_DB || "jobs"
+
+const pool = new Pool({
+  user: user,
```

Comment:
user, host, password... And also port + database can all be changed to format keyAndValue1, keyAndValue2

so it will be like:
const pool = new Pool({
user,
host,
database,
password,
port
});